banyan/src/zstd_dag_cbor_seq.rs

```diff
@@ -102,6 +102,8 @@ impl ZstdDagCborSeq {
                 full = true;
                 break;
             }
+            // scrape links from the new item
+            scrape_links(bytes.as_ref(), &mut links)?;
```

Comment:
This is the actual bugfix. Turns out that if you don't call scrape_links, links don't get scraped.